Frontend/pointmap-frontend/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "pointmap-frontend",
+  "version": "1.0.0",
+  "description": "A Vue.js project",
+  "author": "ChrisMeyer7088 <chris_meyer_2@hotmail.com>",
+  "private": true,
+  "scripts": {
+    "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
+    "start": "npm run dev",
```

Comment:
None of the scripts seem to work on my local machine, can start the vue application